examples/metrics/kafka-connect-metrics.yaml

```diff
@@ -8,7 +8,22 @@ spec:
   version: 2.6.0
   replicas: 1
   bootstrapServers: my-cluster-kafka-bootstrap:9092
-  metrics:
+  metricsConfig:
+    type: jmxPrometheusExporter
+    valueFrom:
+      configMapKeyRef:
+        name: connect-metrics
+        key: metrics-config.yml
+
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: connect-metrics
+  labels:
+    app: strimzi
+data:
+  metrics-config.yml: |
```

Comment:
A comment with a reference to the docs for the jmx exporter config would not hurt.